src/gui/updater/updatefinisheddialog.ui

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>UpdateFinishedDialog</class>
+ <widget class="QDialog" name="UpdateFinishedDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>368</width>
+    <height>190</height>
+   </rect>
+  </property>
+  <property name="sizePolicy">
+   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
+    <horstretch>0</horstretch>
+    <verstretch>0</verstretch>
+   </sizepolicy>
+  </property>
+  <property name="windowTitle">
+   <string>Restart required</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
+   <property name="spacing">
+    <number>12</number>
+   </property>
+   <property name="sizeConstraint">
+    <enum>QLayout::SetFixedSize</enum>
+   </property>
+   <item>
+    <widget class="QLabel" name="label">
+     <property name="sizePolicy">
+      <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
+       <horstretch>0</horstretch>
+       <verstretch>0</verstretch>
+      </sizepolicy>
+     </property>
+     <property name="minimumSize">
+      <size>
+       <width>350</width>
+       <height>0</height>
+      </size>
+     </property>
+     <property name="maximumSize">
+      <size>
+       <width>350</width>
+       <height>16777215</height>
+      </size>
+     </property>
+     <property name="text">
+      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Update finished successfully.&lt;/p&gt;&lt;p&gt;Do you want to restart the client now?&lt;/p&gt;&lt;p&gt;Note: if you dismiss this dialog, you need to restart the client manually in order for the update to take effect.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
I'm not sure about the wording. I'll check a couple of applications to see how the wording is done there.

Reply:
I also am not too happy with this wording. Suggestions welcome.

Reply:
@tbsbdr @EParzefall for help? ⬆️

Reply:
Proposal: I'd only talk about the happy path (=restart) and keep it short:


---


### Update installed successfully
Restart now to finish installing the update.
`Restart now` `Restart later`


---

Reply:
My suggestion: Restart now for changes to take effect (otherwise a manual restart is necessary later).

Reply:
These kind of messages are always in passive voice, meaning, the "you" is always left out. (This is in contrast to e.g. a FAQ, where the user is directly addressed.) I'd go for the second suggestion, because it makes it clear that the user needs to take an action if the restart is postponed.

Reply:
^

Reply:
![screenshot_2023-11-08_14-57-23](https://github.com/owncloud/client/assets/80399010/e0d07d9a-d60d-4f21-9e2d-528bb0ff78da)

Reply:
We need to clarify a that we are talking about an application restart, not a system one.
The second one is overly complex.

Reply:
The first one is nice and short. Maybe phrase the later button as "Restart later manually"?

Reply:
The buttons' text is very long (and it won't get better when translated), this is why I prefer to use "yes/no". I don't see the complexity either. Let's try, though.

Reply:
![screenshot_2023-11-08_15-29-54](https://github.com/owncloud/client/assets/80399010/7b315579-e49e-436e-bc8c-9bcc276f4046)

Reply:
how about `application restart required` also for the header?

Reply:
![screenshot_2023-11-08_15-44-52](https://github.com/owncloud/client/assets/80399010/2579691b-0156-4e64-9d23-e9ccb95ed67b)